packages/client/src/encryption/GroupKeyManager.ts

```diff
@@ -0,0 +1,74 @@
+import { LitProtocolFacade } from './LitProtocolFacade'
+import { inject, Lifecycle, scoped } from 'tsyringe'
+import { GroupKeyStore } from './GroupKeyStore'
+import { GroupKey } from './GroupKey'
+import { StreamID, StreamPartID, StreamPartIDUtils } from '@streamr/protocol'
+import { EthereumAddress, waitForEvent } from '@streamr/utils'
+import { SubscriberKeyExchange } from './SubscriberKeyExchange'
+import { ConfigInjectionToken, StrictStreamrClientConfig } from '../Config'
+import { StreamrClientEventEmitter } from '../events'
+import { DestroySignal } from '../DestroySignal'
+import crypto from 'crypto'
+import { uuid } from '../utils/uuid'
+
+@scoped(Lifecycle.ContainerScoped)
+export class GroupKeyManager {
+    constructor(
+        @inject(GroupKeyStore) private readonly groupKeyStore: GroupKeyStore,
```

Comment:
The `@inject` annotations are not needed when we inject `scoped` classes. Could remove the annotations from lines `17`-`21` (but not `22`) to highlight that the injection of `config` field is different compared to the other fields.